package.json

```diff
@@ -24,17 +24,17 @@
     "graphql": "0.10.5",
     "graphql-tools": "1.1.0",
     "isomorphic-fetch": "2.2.1",
-    "nock": "^9.0.14",
+    "nock": "9.0.14",
```

Comment:
@michaelmurray6298 please remove this line as you already have `nock` installed as a dev dependency, thanks!